invenio_accounts/profiles/schemas.py

```diff
@@ -43,7 +44,8 @@ class UserProfileSchema(Schema):
     """The default user profile schema."""
 
     full_name = fields.String()
-    affiliations = fields.String()
```

Comment:
This is not correct. We use sanitised HTML when we want to store and output HTML, and string when we just want text.